### PR TITLE
feat(news-articles):make articles in main clickable to expand them an…

### LINKED ISSUE #25: Expand news article animation
**Is your feature request related to a problem? Please describe.**
The articles need to give more detail

**Describe the solution you'd like**
To click the articles and then enlarge them to give more detail

### PR DESCRIPTION
**Pull request by Lap343**
…d have a x button to contract them

## Changes
1. Make articles in main clickable to expand 
2. Added a X button to the expanded article to contract them

## Purpose
To give the reader more details when they click on an article

Closes #25

### CODE REVIEW

**Review by connielion — APPROVED** [review]
Looks good!

**Review by BlackCubes — APPROVED** [review]
JOOICY! But noice!

**Review by krferrell — APPROVED** [review]
Solid work. No issues that I can see.

File: modern-news-app/src/components/Main.tsx [hunk 1]
```diff
@@ -26,6 +26,53 @@ const TitleStyled = styled.div`
...
+            display: block;
+            font-size: 1.8em;
+            font-weight: 400;
+            text-decoration: none;
+            color: white;
+            padding-bottom: 0.5em;
+        }
+    }
+`
+
```
Comment (JorgeLVilla) [inline modern-news-app/src/components/Main.tsx:75]: Cleeeean.

**Review by JorgeLVilla — APPROVED** [review]
Great job Luis!

**Pull request merged**